src/app/pages/IndexPage/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const IndexPage = () => {
+  return (
+    <main>
```

Comment:
```suggestion
    <main role="main">
```

Reply:
Can we also add the `h1` as a part of the skeleton?

```suggestion
    <h1 id="content">Index Page</h1>
```